vizro-core/examples/kpi/utils/_helper.py

```diff
@@ -38,18 +40,89 @@ def clean_data_and_add_columns(data: pd.DataFrame):
     data["Company response - detailed"] = data["Company response - detailed"].replace("Closed", "Closed without relief")
     data["State"] = data["State"].replace("UNITED STATES MINOR OUTLYING ISLANDS", "UM")
     data["State"] = fill_na_with_random(data, "State")
+    data["Consumer disputed?"] = data["Consumer disputed?"].fillna("No")
 
     # Convert to correct data type
     data["Date Received"] = pd.to_datetime(data["Date Received"], format="%m/%d/%y").dt.strftime("%Y-%m-%d")
-    data["Date Submitted"] = pd.to_datetime(data["Date Submitted"], format="%m/%d/%y").dt.strftime("%Y-%m-%d")
 
     # Create additional columns
-    data["Year-Month Received"] = pd.to_datetime(data["Date Received"], format="%Y-%m-%d").dt.strftime("%Y-%m")
+    data["Month"] = pd.to_datetime(data["Date Received"], format="%Y-%m-%d").dt.strftime("%m")
+    data["Year"] = pd.to_datetime(data["Date Received"], format="%Y-%m-%d").dt.strftime("%Y")
     data["Region"] = data["State"].map(REGION_MAPPING)
     data["Company response"] = np.where(
         data["Company response - detailed"].str.contains("Closed"), "Closed", data["Company response - detailed"]
     )
     data["Company response - Closed"] = np.where(
         data["Company response - detailed"].str.contains("Closed"), data["Company response - detailed"], "Not closed"
     )
+
+    # Filter 2018 and 2019 only
+    data = data[(data["Year"].isin(["2018", "2019"]))]
```

Comment:
The page loading process will be faster when we merge this PR -> https://github.com/mckinsey/vizro/pull/644. 

Therefore, it is likely that other years can be included in the table if it brings some important information.

Reply:
I thought so, but I think for this demo it doesn't matter anymore :) I am also only adding two years as otherwise the homepage will loose a lot of horizontal space that would be reserved for a single filter 😅  For the next demos, for sure though!